src/com/redhat/ci/provisioner/ProvisioningConfig.groovy

```diff
@@ -4,16 +4,20 @@ package com.redhat.ci.provisioner
  * Configuration needed to provision resources with a Provisioner.
  */
 class ProvisioningConfig {
+    private static final String RELEASE_VERSION = 'v1.2.1'
```

Comment:
1.2.2?

Reply:
Since no provisioner image v1.2.2 exists in openshift it's probably best to wait to update this just prior to the release.